lesson33/src/forgotpassword.html

```diff
@@ -21,14 +21,14 @@
       <nav class="drawer-menu" id="js-drawer-menu" aria-hidden="true">
         <ul class="drawer-menu__nav-list">
           <li class="drawer-menu__nav-item">
-            <a href="./login.html">
+            <a href="./login.html" class="drawer-menu__nav-link">
```

Comment:
Q
Is there any reason not to add js-prefix?

Reply:
Thanks for the review!

Sorry, this is an oversight...I have modified it by adding js-prefix since the element is handled by JavaScript.
[9fb59cd](https://github.com/sae-github/JS_lesson/pull/41/commits/9fb59cdad101bd97ae4aca238a86619023def0f3)

Reply:
Sorry for being too NITS.
Thanks for the fix!